src/driver/AmdCompiler.cpp

```diff
@@ -319,7 +320,7 @@ class AMDGPUCompiler : public Compiler {
   void AddCommonArgs(std::vector<const char*>& args);
   void FilterArgs(ArgStringList& args);
   bool InvokeDriver(ArrayRef<const char*> args);
-  bool InvokeLLVMLink(ArrayRef<const char*> args);
+  bool InvokeLinker(ArrayRef<const char*> args, const std::string& sLinker);
```

Comment:
I guess I object overloading them even more. I cannot see why a single method should serve two completely different tasks just because both accidentally called "link".